.github/workflows/pytorch181_gpu.yml

```diff
@@ -1,25 +0,0 @@
-name: pytorch1_8_1-cuda10_2
-on:
```

Comment:
Can we add a CI for 'pytorch1_8_1-cu113` or other version of cuda 11?

Reply:
Of cause but not in this PR?

Reply:
https://github.com/alibaba/BladeDISC/issues/513